test/App/Fossa/LicenseScannerSpec.hs

```diff
@@ -0,0 +1,75 @@
+module App.Fossa.LicenseScannerSpec (spec) where
```

Comment:
Will you please add an integration test case for the recursive archive expansion? I think the code is good but by definition it's finicky enough that it may be easy to subtly break later.